client/components/focusable/index.jsx

```diff
@@ -30,17 +27,15 @@ class Focusable extends Component {
 	};
 
 	render = () => {
-		const { children, className } = this.props;
-		const omitProps = [ 'children', 'className', 'onClick', 'onKeyDown', 'role', 'tabIndex' ];
-		const props = omit( this.props, omitProps );
+		const { children, className, ...passProps } = this.props;
 		return (
 			<div
+				{ ...passProps }
```

Comment:
Rather than `omit` which is considered [costly](https://lodash.com/docs/4.17.10#omit), we can pass the props here then let them be overwritten by our statically provided props which come after.

Reply:
Alternatively, you could destructure them away on L30.